src/libcore/array/mod.rs

```diff
@@ -202,6 +204,114 @@ where
     }
 }
 
+/// Return Error of the FromIterator impl for array
+#[unstable(feature = "array_from_iter_impl", issue = "none")]
+#[derive(Debug)]
```

Comment:
it might be better to manually implement `Debug` using `Self::as_slice`. MaybeUninit's debug impl is kinda useless.